src/model_data_sweep.yaml

```diff
@@ -0,0 +1,26 @@
+name: Christoph model grid
+method: grid
+parameters:
+  model:
+    value: Christoph_CNN
+  optim:
+    value: christoph
+  optim.epochs:
+    value: 100
+  data:
+    values:
+      - defaults
+      - icml2013
+      - orcasound
+  preprocessors:
+    value: _preprocessors_sliding_window
+  data.train_dataset.augmentations_p:
+    value: 0
```

Comment:
There are no augmentatios on christophs model? weird

Reply:
I seem to remember that was the winner in the seep, but not sure.